composer.json

```diff
@@ -25,35 +25,35 @@
         }
     ],
     "require": {
-        "php": ">=7.3.0,<8.1.0",
+        "php": ">=7.4.3,<8.1.0",
         "ext-json": "*",
-        "daverandom/exceptional-json": "^1.0.4",
-        "guzzlehttp/guzzle": "^6.5.2 || ^7.0",
-        "monolog/monolog": "^1.25.3 || ^2.0.2",
+        "guzzlehttp/guzzle": "^7.4.0",
+        "psr/log": "^1.1.4 || ^2.0.0 || ^3.0.0",
         "psr/simple-cache": "^1.0.1",
-        "roave/doctrine-simplecache": "^2.2.0",
-        "symfony/console": "^3.4.36 || ^4.4.2 || ^5.0.2",
-        "symfony/filesystem": "^3.4.36 || ^4.4.2 || ^5.0.2"
+        "symfony/filesystem": "^5.4.0"
     },
     "require-dev": {
-        "ergebnis/composer-normalize": "^2.2.2",
-        "friendsofphp/php-cs-fixer": "^2.16.1",
-        "mikey179/vfsstream": "^1.6.8",
-        "pepakriz/phpstan-exception-rules": "^0.11.1",
-        "phpstan/phpstan": "^0.12.9",
-        "phpstan/phpstan-beberlei-assert": "^0.12.2",
-        "phpstan/phpstan-deprecation-rules": "^0.12.2",
-        "phpstan/phpstan-phpunit": "^0.12.6",
-        "phpstan/phpstan-strict-rules": "^0.12.2",
-        "phpunit/phpunit": "^9.5.0"
+        "doctrine/coding-standard": "^9.0.0",
+        "league/flysystem": "^2.3.2",
+        "matthiasmullie/scrapbook": "^1.4.8",
+        "mikey179/vfsstream": "^v1.6.10",
+        "monolog/monolog": "^2.3.5",
+        "phpstan/extension-installer": "^1.1.0",
+        "phpstan/phpstan": "^1.2.0",
+        "phpstan/phpstan-beberlei-assert": "^1.0.0",
+        "phpstan/phpstan-deprecation-rules": "^1.0.0",
+        "phpstan/phpstan-phpunit": "^1.0.0",
+        "phpunit/phpunit": "^9.5.10",
+        "symfony/console": "^5.4.0"
     },
     "suggest": {
-        "ext-curl": "to use curl requests to get the ini file"
+        "ext-curl": "to use curl requests to get the ini file",
+        "league/flysystem": "Filesystem cache, Required to use the console commands",
+        "matthiasmullie/scrapbook": "Caching Lib, Required to use the console commands",
+        "monolog/monolog": "Logging Lib, Required to use the console commands",
+        "symfony/console": "Required to use the console commands"
```

Comment:
I would say `symfony/console` is actually a required dependency, rather than a `suggest`

Reply:
I you dont want to use the console commands, then you dont need it.

Reply:
It's difficult to analyse code where dependencies are optional; the CLI is a large part of how you use it IMO, and few would object to symfony/console IMO, even if they didn't already have it heh